vendor/github.com/docker/distribution/registry/api/errcode/errors.go

```diff
@@ -1,267 +0,0 @@
-package errcode
```

Comment:
I think these became no longer necessary after commit https://github.com/containerd/containerd/pull/3728/commits/901bcb2231466229d27aee8d38a6e2fcdc95015e (https://github.com/containerd/containerd/pull/3728)

Could it be that the vendor validation doesn't run unless `vendor.conf` was modified? If so, that would explain why CI didn't catch the problem (ran into a similar issue in moby/moby, and have a PR there to _force_ the vendor check, even if `vendor.conf` wasn't updated); https://github.com/moby/moby/pull/40050

Reply:
sure~